The player is contacted by Republic Intel after making contact with the Hai (#7001)
endless-sky/endless-sky: data/human/human missions.txt

				decline

mission "Block Hai Wormhole Warning"
	landing
	invisible
	to offer
		or
			has "First Contact: Hai"
			has "First Contact: Unfettered"
		has "chosen sides"
		not "main plot completed"
	on offer
		fail

mission "Hai Wormhole Warning"
	minor
	source
		government "Republic"
		not attributes "deep"
	to offer
		or
			has "First Contact: Hai"
			has "First Contact: Unfettered"
		not "Block Hai Wormhole Warning: offered"
		not "Wanderers: Alpha Surveillance E: offered"
	on offer
		log "Was pulled aside by a Navy officer and told to keep the wormhole to the Hai a secret. The Republic doesn't want the knowledge of the wormhole to be widespread, as that could cause chaos for both the Republic and the Hai."
		conversation

> 			`Upon entering the spaceport, you're stopped by Navy officer. "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on a landing pad behind you.`

Reviewer: ```suggestion
			`Upon entering the spaceport, you're stopped by a Navy officer. "Captain <last>, I'd like to speak with you for a few minutes... In private." He motions toward a Frigate on a landing pad behind you.`
```

Author: Edited your suggestion: I'd like to keep the dialog as three separate sentences.

			choice
				`	"What about?"`

> 					goto what

Author: ```suggestion
					goto what
				`	"Okay."`
					goto frigate
```

			choice
				`	"It is. Is that a problem?"`
					goto yes
				`	"Perhaps. Why do you ask?"`
					goto maybe
				`	"Aliens? I don't know of any aliens up there."`
			`	"Don't play coy with me, Captain. We are aware that you've made contact with the Hai.`
				goto secret
			label maybe
			`	The officer chuckles. "I think the answer you're looking for is 'Yes,' not 'Perhaps.' We are aware that you've made contact with the Hai.`
				goto secret
			label yes
			`	"There is no problem, Captain, but we are aware that you've made contact with the Hai.`
			label secret
			`	"But you may not be aware of the arrangement that the Republic has with the Hai. The Republic works hard to ensure that the existence of the Hai remains an open secret; we do not disallow captains such as yourself from stumbling upon them, but we do not allow this existence to be widely shared."`
			choice
				`	"Why are you telling me this?"`
					goto why
				`	"So you're threatening me to stay quiet?"`
			`	"No threats, only a... warning, so to say.`
				goto chaos
			label why
			`	"I'm telling you this as something of a... warning, so to say.`
			label chaos

> 			`	"I'm sure that you can imagine that should the existence of the Hai become public, that would cause a great deal of chaos both for us and for the Hai, chaos that could easily end badly, particularly for the one who caused it. So it would be in everyone's interest if you kept this as our little secret."`

Reviewer: ```suggestion
			`	"I'm sure that you can imagine that should the existence of the Hai become public, it would cause a great deal of chaos for both us and for the Hai, chaos that could easily end badly, particularly for the one who caused it. So it would be in everyone's interest if you kept this as our little secret."`
```

Author: Edited your suggestion: kept "both" where it is and added a comma before it.

			choice
				`	"Alright. I won't let anyone know."`
					goto ok
				`	"And if I don't?"`

> 			`	The officer leans back in his chair. "As I said, I'm giving you a warning that things could end badly for you, so you may want to think twice about revealing such information. Now if you'll please depart my ship so that I can be on my way."`

Reviewer: ```suggestion
			`	The officer leans back in his chair. "As I said, I'm giving you a warning that things could end badly for you, so you may want to think twice about revealing such information. Now, if you'll please depart my ship so that I can be on my way."`
```

				goto end
			label ok

> 			`	"That's good to hear," the officer responds. "Thank you for your cooperation. Now if you'll please depart my ship so that I can be on my way."`

Reviewer: ```suggestion
			`	"That's good to hear," the officer responds. "Thank you for your cooperation. Now, if you'll please depart my ship so that I can be on my way."`
```

			label end

> 			`	The officer leads you out of the ship and closes the doors behind you. As you make your way back to the spaceport, you turn back and see the Frigate taking off, without ever having seen more crew board the ship.`

Reviewer: ```suggestion
			`	The officer leads you out of the ship and closes the doors behind you. As you make your way back to the spaceport, you turn back and see the Frigate taking off, without ever having seen any crew board the ship.`
```

			has "First Contact: Unfettered"
		has "chosen sides"
		not "main plot completed"
	on offer
		fail

mission "Hai Wormhole Warning"
	minor
	source
		government "Republic"
		not attributes "deep"
	to offer
		or
			has "First Contact: Hai"
			has "First Contact: Unfettered"
		not "Block Hai Wormhole Warning: offered"
		not "Wanderers: Alpha Surveillance E: offered"
	on offer
		log "Was pulled aside by a Navy officer and told to keep the wormhole to the Hai a secret. The Republic doesn't want the knowledge of the wormhole to be widespread, as that could cause chaos for both the Republic and the Hai."
		conversation
			`Upon entering the spaceport, you're stopped by a Navy officer. "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on a landing pad behind you.`
			choice
				`	"What about?"`
					goto what
				`	"Okay."`
					goto frigate
				`	"Sorry, I'm really busy right now."`
			`	"No, I'm sorry, but this isn't a matter of whether you're busy or not right now. Come with me."`
				goto frigate
			label what
			`	"I did say in private, did I not?" he says, once again motioning to the Frigate.`
			label frigate

> 			`	The man puts a hand on your shoulder and begins walking you toward the Frigate. As you approach, you note that the man's uniform doesn't have any Naval markings that you recognize.`

Reviewer 2: `you note that the man's uniform doesn't have any Naval markings that you recognize`
Pray tell, how then did you know he was a naval officer?

Author: This same language is used in the Pact Questioning mission, i.e. no recognizable markings but the character is referred to as an officer.

Reviewer 2: A project for a later time then, noted.

> 			`	You enter the Frigate with the officer, and despite the decent crew requirement of a Frigate, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`

Reviewer 3: ```suggestion
			`	You enter the Frigate with the officer, and despite the decent crew requirement of that ship, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`
```

Author: How about flipping which "Frigate" we replace, since the previous line already notes that it's a Frigate.
```suggestion
			`	You enter the ship with the officer, and despite the decent crew requirement of a Frigate, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`
```

			label questioning
			`	"So what is it that you brought me here for?" you ask.`
			`	The officer folds his hands together. "It has come to the Republic's attention that you've made contact with an alien species located to the north of the Republic. Is this correct?"`
			choice
				`	"It is. Is that a problem?"`
					goto yes
				`	"Perhaps. Why do you ask?"`
					goto maybe
				`	"Aliens? I don't know of any aliens up there."`
			`	"Don't play coy with me, Captain. We are aware that you've made contact with the Hai.`
				goto secret
			label maybe
			`	The officer chuckles. "I think the answer you're looking for is 'Yes,' not 'Perhaps.' We are aware that you've made contact with the Hai.`
				goto secret
			label yes
			`	"There is no problem, Captain, but we are aware that you've made contact with the Hai.`
			label secret
			`	"But you may not be aware of the arrangement that the Republic has with the Hai. The Republic works hard to ensure that the existence of the Hai remains an open secret; we do not disallow captains such as yourself from stumbling upon them, but we do not allow this existence to be widely shared."`

> 			choice

Reviewer 4: In this choice (and also in the next one), I would like to have the option to ask for more details. "Can you tell me a bit more about this arrangement? Who were involved? Where and when was it decided?"

Author: I only added an extra choice here. How does that look?

Reviewer 4: Looks good!

				`	"Why are you telling me this?"`
					goto why
				`	"So you're threatening me to stay quiet?"`
			`	"No threats, only a... warning, so to say.`
				goto chaos
			label why
			`	"I'm telling you this as something of a... warning, so to say.`
			label chaos
			`	"I'm sure that you can imagine that should the existence of the Hai become public, it would cause a great deal of chaos, both for us and for the Hai, chaos that could easily end badly, particularly for the one who caused it. So it would be in everyone's interest if you kept this as our little secret."`
			choice
				`	"Alright. I won't let anyone know."`
					goto ok
				`	"And if I don't?"`

> 			`	The officer leans back in his chair. "As I said, I'm giving you a warning that things could end badly for you, so you may want to think twice about revealing such information. Now, if you'll please depart my ship so that I can be on my way."`

Reviewer 2: I feel like this line could serve to imply a little more. Right now, this is basically just the Republic issuing a veiled threat, even if it's presented as a warning. With a little tweaking though...
```suggestion
			`	The officer leans back in his chair. "As I said, I'm giving you a warning that things could end badly for you. We work hard to maintain it as an open secret but we also work hard to ensure the safety of our citizens. However, we are not the only ones who work hard to maintain the secret, so you may want to think twice about revealing such information. Now, if you'll please depart my ship so that I can be on my way."`
```

Reviewer 2: We could do that, though that on its own to me feels like it has a connotation of there being a singular, other, more dangerous power working at it - whereas by at least calling out that they look after citizen safety, that implies that they are the biggest (or nearly so) of multiple powers working at it.

The first part can definitely be dropped though, I put him restating it just for emphasis but it's not necessary.

				goto end
			label ok

> 			`	"That's good to hear," the officer responds. "Thank you for your cooperation. Now, if you'll please depart my ship so that I can be on my way."`

Reviewer 2: This is a soft suggestion, the phrasing leans slightly more into the implicit suggestion that there are multiple party's interests represented here. 
(Also has a bit of a "good, you're one less person who's safety we might have to consider *after* you do something dumb")
```suggestion
			`	"That's good to hear," the officer responds. "Your cooperation is appreciated. Now, if you'll please depart my ship so that I can be on my way."`
```

		conversation
			`You receive a payment of <payment> from the down-on-his-luck laborer that you transported a while back, along with a message. "Hello <last>. Not sure if you remember me. I'm the one that had to deal with my boss bad-talking me to everyone on my home planet. I've been doing better, but my new boss keeps trying to rip me off and make me come in for overtime." He goes into detail about the sordid working conditions in his new job, but you stop reading after the third paragraph.`
				decline

mission "Block Hai Wormhole Warning"
	landing
	invisible
	to offer
		or
			has "First Contact: Hai"
			has "First Contact: Unfettered"
		has "chosen sides"
		not "main plot completed"
	on offer
		fail

mission "Hai Wormhole Warning"
	minor
	source
		government "Republic"
		not attributes "deep"
	to offer
		or
			has "First Contact: Hai"
			has "First Contact: Unfettered"
		not "Block Hai Wormhole Warning: offered"
		not "Wanderers: Alpha Surveillance E: offered"
	on offer

> 		log "Was pulled aside by a Navy officer and told to keep the wormhole to the Hai a secret. The Republic doesn't want the knowledge of the wormhole to be widespread, as that could cause chaos for both the Republic and the Hai."

Reviewer 5: A Navy officer? Is Republic Intel not separate from the Navy? Does <first> <last> not understand the distinction or do they not realise that this person isn't actually a part of the Navy (using a Frigate could certainly lead to confusion there)?

Author: As I said to Grey, this is the language used elsewhere for this exact same scenario.

		conversation
			`Upon entering the spaceport, you're stopped by a Navy officer. "Captain <last>. I'd like to speak with you for a few minutes. In private." He motions toward a Frigate on a landing pad behind you.`
			choice
				`	"What about?"`
					goto what
				`	"Okay."`
					goto frigate
				`	"Sorry, I'm really busy right now."`
			`	"No, I'm sorry, but this isn't a matter of whether you're busy or not right now. Come with me."`
				goto frigate
			label what
			`	"I did say in private, did I not?" he says, once again motioning to the Frigate.`
			label frigate
			`	The man puts a hand on your shoulder and begins walking you toward the ship. As you approach, you note that the man's uniform doesn't have any Naval markings that you recognize.`

> 			`	You enter the ship with the officer, and despite the decent crew requirement of a Frigate, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`

Reviewer 5: We know we're on the ship, don't need to tell us again.
```suggestion
			`	You enter the ship with the officer, and despite the decent crew requirement of a Frigate, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area. "Please, have a seat," he says.`
```

			choice
				`	(Take a seat.)`
					goto seat
				`	"No, I like to stand."`
			`	"Suit yourself," the officer says with a shrug while taking a seat himself.`
				goto questioning
			label seat
			`	You choose a table to sit at and the officer sits across from you.`
			label questioning
			`	"So what is it that you brought me here for?" you ask.`

> 			`	The officer folds his hands together. "It has come to the Republic's attention that you've made contact with an alien species located to the north of the Republic. Is this correct?"`

Reviewer 5: Does he have to always refer to the Republic in the third person and only by name, never pronouns?
```suggestion
			`	The officer folds his hands together. "It has come to the Republic's attention that you've made contact with an alien species located to the north of it. Is this correct?"`
```
```suggestion
			`	The officer folds his hands together. "It has come to our attention that you've made contact with an alien species located to the north of the Republic. Is this correct?"`
```

			choice
				`	"It is. Is that a problem?"`
					goto yes
				`	"Perhaps. Why do you ask?"`
					goto maybe
				`	"Aliens? I don't know of any aliens up there."`
			`	"Don't play coy with me, <last>. We are aware that you've made contact with the Hai.`
				goto secret
			label maybe
			`	The officer chuckles. "I think the answer you're looking for is 'Yes,' not 'Perhaps.' We are aware that you've made contact with the Hai.`
				goto secret
			label yes

> 			`	"There is no problem, <last>, but we are aware that you've made contact with the Hai.`

Reviewer 5: Following the other two options, this phrase makes sense, but here it seems a bit odd.
"The Republic has learned that you've made contact with some alients to the north. Is this correct?"
"It is, is that a problem?"
"No problem, but we are aware that you've made contact with the Hai."
Dropping the question in our answer:
"It is."
"We are aware that you've made contact with the Hai."
My response would be something like:
"Yeah, you already told me that, and I just told you that I have."
Maybe if he thought we might be talking about the Wanderers, but there's no indication anywhere else that that's the case.
```suggestion
			`	"There is no problem, <last>, I just want to make sure you also know about an arrangement the Republic has with the Hai.`
			`	"The Republic works hard to ensure that the existence of the Hai remains an open secret; we do not disallow captains such as yourself from stumbling upon them, but we will not allow this existence to be widely shared."
				goto question
```

			label secret

> 			`	"But you may not be aware of the arrangement that the Republic has with the Hai. The Republic works hard to ensure that the existence of the Hai remains an open secret; we do not disallow captains such as yourself from stumbling upon them, but we will not allow this existence to be widely shared."`

Reviewer 5: ```suggestion
			`	"But you may not be aware of the arrangement that the Republic has with the Hai. The Republic works hard to ensure that the existence of the Hai remains an open secret; we do not disallow captains such as yourself from stumbling upon them, but we will not allow this knowledge to be widely shared."`
```
```suggestion
			`	"But you may not be aware of the arrangement that the Republic has with the Hai. The Republic works hard to ensure that the existence of the Hai remains an open secret; we do not disallow captains such as yourself from stumbling upon them, but we will not allow this existence to be widely publicized."`
```

> 			choice

Reviewer 5: ```suggestion
			label question
			choice
```

				`	"Why are you telling me this?"`
					goto why

> 				`	"An arrangement between the Republic and Hai? When did that happen?"`

Reviewer 3: ```suggestion
				`	"An arrangement between the Republic and the Hai? When did that happen?"`
```

Author: Seems like a personal preference change to me, and I prefer how it is now.

Reviewer 3: Okay, not a big deal either way.

					goto arrangement
				`	"So you're threatening me to stay quiet?"`
			`	"No threats, only a... warning, so to say.`
				goto chaos
			label why
			`	"I'm telling you this as something of a... warning, so to say.`
				goto chaos
			label arrangement
			`	"The details of the arrangement are none of your concern. All you need to know is that we provide a... warning, so to say, to those who know of the Hai that they keep them secret.`
			label chaos

> 			`	"I'm sure that you can imagine that should the existence of the Hai become public, it would cause a great deal of chaos, both for us and for the Hai, chaos that could easily end badly, particularly for the one who caused it. So it would be in everyone's interest if you kept this as our little secret."`

Reviewer 5: ```suggestion
			`	"I'm sure you can imagine that, should the existence of the Hai become public, it would cause a great deal of chaos, both for us and for the Hai. Chaos that could easily end badly, particularly for the one who caused it. So it would be in everyone's interest if you kept this as our little secret."`
```
